src/persisto.js

```diff
@@ -521,8 +521,10 @@
      */
     writeToForm: function(form, options) {
       var $form = $(form);
+      self = this;
```

Comment:
this line breaks the tests. It should be `var self = this`. 
Please join this with the prev. `var` statement using a comma.